weio/fast_output_file.py

```diff
@@ -212,6 +264,81 @@ def unit(s):
         units = [unit(c) for c in self.data.columns]
         return units
 
+    def _readAll(self):
+        """Read all data after header in streaming mode."""
+        import numpy as np
+
+        if self._fid is None:
+            raise RuntimeError("No open file handle. Use streaming=True with context manager.")
+
+        ext = os.path.splitext(self.filename.lower())[1]
+
+        # Read remaining data from open file handle
+        if self['binary']:
+            # Binary file - use shared data reading function
+            try:
+                # Build info dict from stored metadata
+                info = {
+                    'FileID': self['_FileID'],
+                    'NumOutChans': self['_NumOutChans'],
+                    'NT': self['_NT'],
+                    'ColScl': self['_ColScl'],
+                    'ColOff': self['_ColOff']
+                }
+                if self['_FileID'] == FileFmtID_WithTime:
+                    info['TimeScl'] = self['_TimeScl']
+                    info['TimeOff'] = self['_TimeOff']
+                else:
+                    info['TimeOut1'] = self['_TimeOut1']
+                    info['TimeIncr'] = self['_TimeIncr']
+

```

Comment:
no need to build "info" if is stored into info_binary after reading the headers.